common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -230,9 +230,11 @@ def has_support(self, view, functionality):
         Returns whether the given view has support for the given functionality.
         """
         if functionality == "multi_device":
-            return self.lcp.has_multi_device_support
-        else:
-            return False
+            return all(
+                responsetypes.registry.get_class_for_tag(tag).multi_device_support
+                for tag in self.problem_types
```

Comment:
Is this checking only the components that are used in this instance of the capa problem?

For example, if there's a composite capa problem with just checkbox and doesn't include other capa components, then we only want to check multi-device-support for only the checkbox.